File: CS/CustomAutoFilterRowFiltering/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;

namespace CustomAutoFilterRowFiltering
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            myGridView1.OptionsView.ShowAutoFilterRow = true;
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'northwindDataSet.Orders' table. You can move, or remove it, as needed.
            this.ordersTableAdapter.Fill(this.northwindDataSet.Orders);
            

        }
    }
}

File: VB/CustomAutoFilterRowFiltering/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Windows.Forms

Namespace CustomAutoFilterRowFiltering
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			myGridView1.OptionsView.ShowAutoFilterRow = True
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'northwindDataSet.Orders' table. You can move, or remove it, as needed.
			Me.ordersTableAdapter.Fill(Me.northwindDataSet.Orders)


		End Sub
	End Class
End Namespace
